tests/networkpolicy_test.go

```diff
@@ -181,16 +181,22 @@ func createVMICirros(virtClient kubecli.KubevirtClient, namespace string, labels
 	return vmi
 }
 
-func checkPingSuccess(fromVmi *v1.VirtualMachineInstance, toIp string) {
-	Expect(tests.PingFromVMConsole(fromVmi, toIp, "-w 8")).To(Succeed())
+func checkPingSuccess(fromVmi, toVmi *v1.VirtualMachineInstance) {
+	ExpectWithOffset(1, toVmi.Status.Interfaces[0].IPs).NotTo(BeEmpty())
+	for _, ip := range toVmi.Status.Interfaces[0].IPs {
+		Expect(tests.PingFromVMConsole(fromVmi, ip, "-w 8")).To(Succeed())
```

Comment:
If you do pass extra args to `PingFromVMConsole` it will override override the "-c 1" are you ok with that ? 

We are going to replace Eventually with "-w 8" right ?

Reply:
ye we dont want here eventually and we are ok to override -c 1 
we do want to see couple pings pass
we are limited by 10 seconds max unless we add the 10 seconds as a parameter

anyhow this should be discussed on 4045 i think (and indeed we discussed it there), since it belong there
https://github.com/kubevirt/kubevirt/pull/4045#discussion_r481789054
(see this please and the follows)